docs/developer/extending/plugins.mdx

```diff
@@ -4,20 +4,15 @@ title: Plugins
 
 ## Introduction
 
-Saleor arrives with some plugins already implemented by default. These plugins are located in `saleor.plugins`.
-To provide the `PluginsManager` with a list of enabled plugins, include the Python plugin path in the `settings.PLUGINS` list.
+Plugins offer a mechanism to extend Saleor's Python core with new functionality. They are loaded by the core process and use predefined callbacks to execute custom code in response to certain events.
 
-## Plugins manager
-
-The `PluginsManager` is located in the `saleor.plugins.manager`. It is a class responsible for handling all declared plugins and serving a response from them. In case of a non-declared plugin, it serves a default response.
-
-It is possible to overwrite an `PluginsManager` class by implementing it on its own. Saleor will discover the manager class by taking the declared path from `settings.PLUGINS_MANAGER`.
+Saleor ships with some plugins included, you can find those in the `saleor/plugins/` directory of the core.
```

Comment:
Don't you think that we could also include some notes about `PaymentGateway` here?  They use the `plugin` interface but they are located in a different directory.

Reply:
Good point, done.